ui/src/app/workflows/components/workflow-drawer/workflow-drawer.tsx

```diff
@@ -34,27 +34,29 @@ export class WorkflowDrawer extends React.Component<WorkflowDrawerProps, Workflo
     }
 
     public render() {
-        if (!this.state.workflow) {
+        const wf = this.state.workflow;
+
+        if (!wf) {
             return <Loading />;
         }
-        const wf = this.state.workflow;
+
         return (
             <div className='workflow-drawer'>
-                {!wf.status.message ? null : (
+                {!wf.status || !wf.status.message ? null : (
```

Comment:
```suggestion
                {(!wf.status || !wf.status.message) ? null : (
```
This seems safer? Or at the very lest it's more readable.

Reply:
@simster7 TSLint doesn't like that change:

```
yarn run v1.22.4
$ /Users/dherman/src/argoproj/argo/ui/node_modules/.bin/tslint -p ./src/app

ERROR: /Users/dherman/src/argoproj/argo/ui/src/app/workflows/components/workflow-drawer/workflow-drawer.tsx[45, 18]: Replace `(!wf.status·||·!wf.status.message)` with `!wf.status·||·!wf.status.message`
ERROR: /Users/dherman/src/argoproj/argo/ui/src/app/workflows/components/workflow-drawer/workflow-drawer.tsx[51, 18]: Replace `(!wf.status·||·!wf.status.conditions)` with `!wf.status·||·!wf.status.conditions`
ERROR: /Users/dherman/src/argoproj/argo/ui/src/app/workflows/components/workflow-drawer/workflow-drawer.tsx[59, 18]: Replace `(!wf.status·||·!wf.status.resourcesDuration)` with `!wf.status·||·!wf.status.resourcesDuration`
```